src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1310,7 +1373,7 @@
     <value>Two arrays, {0} and {1}, must be of  the same size.</value>
   </data>
   <data name="Argument_MissingDefaultConstructor" xml:space="preserve">
-    <value>was missing default constructor.</value>
+    <value>Was missing default constructor.</value>
```

Comment:
This looks like an incomplete sentence. This error message should be deleted and replaced with `Arg_NoDefCTorWithoutTypeName` or `Arg_NoDefCTor`.